From 4ee4cadd1e05736fbd9d2ca1da290a970f08606d Mon Sep 17 00:00:00 2001
From: Niko Tyni <ntyni@debian.org>
Date: Sat, 8 May 2010 13:32:27 +0300
Subject: [PATCH] Use $Config{debian_abi} to generate the perlapi-* dependency

As per the upcoming policy change discussed in #579457,
perlapi-$Config{debian_abi} is preferred over perlapi-$Config{version}
where available.
---
 dh_perl |    3 ++-
 1 files changed, 2 insertions(+), 1 deletions(-)

diff --git a/dh_perl b/dh_perl
index d43043f..5739ca9 100755
--- a/dh_perl
+++ b/dh_perl
@@ -128,7 +128,8 @@ foreach my $package (@{$dh{DOPACKAGES}}) {
 			unless $perl eq 'perl-base' && ! length($version);
 
 		# add perlapi-<ver> for XS modules
-		addsubstvar($package, "perl:Depends", "perlapi-$Config{version}")
+		addsubstvar($package, "perl:Depends",
+			"perlapi-" . ($Config{debian_abi} || $Config{version}))
 			if $deps & XS_MODULE;
 	}
 
-- 
1.7.1

